web-console/package.json

```diff
@@ -74,6 +74,7 @@
     "reactflow": "^11.8.0",
     "ts-pattern": "^5.0.5",
     "use-debounce": "^9.0.4",
+    "valibot": "^0.17.0",
     "yaml": "^2.3.1",
     "yup": "^1.2.0",
```

Comment:
is yup still used since you used this new library now?

Reply:
otherwise we can remove it from package.json .. maybe we should ahve a static check that tests for unused but declared dependencies

Reply:
yup is still used elsewhere in the codebase, I intend to gradually replace it